### ISSUE TITLE
Fail to compile the temporal sampler

### ISSUE THREAD

**Issue by eddiegaoo** [amazon-science/tgl#17]
Hi, thanks for sharing the good work.

I am interested in using TGL framework, but I got stuck in the first place when I should compile the C++ implemented temporal sampler. 

And I get the following error:
`error: command '/bin/gcc' failed with exit code 1`

I wonder how to fix this issue. Thanks!

**Comment by tedzhouhk** [comment]
Hi, could you please provide more information? My current guess is that installing python3-dev might solve this problem.

**Comment by eddiegaoo** [comment]
Thanks for the reply. I will try to install python3-dev to see if it fixes my problem.

I checked the bugs last time, it seems the gcc(g++) version in the server I used is outdated (4.8.5). But I have no root authorization to update it. 

Then I try to update it in the miniconda3 virtual environments using the approach described in https://www.zhihu.com/question/56272908, but it seems to throw another different bug described as below:
`x86_64-conda-linux-gnu-cc: fatal error: cannot execute 'cc1plus': execvp: No such file or directory
compilation terminated.
error: command '/home/data/personal_file/envs/miniconda3/envs/tgl/bin/x86_64-conda-linux-gnu-cc' failed with exit code 1`
